doc/glossary.rst

```diff
@@ -209,9 +209,9 @@ General Concepts
         TODO. Mention casting.
 
     duck typing
-        We try to apply [duck
-        typing](https://en.wikipedia.org/wiki/Duck_typing) to determine how to
-        handle some input values (e.g. checking whether a given estimator is a
+        We try to apply `duck typing
```

Comment:
You forgot a back tick

https://15611-843222-gh.circle-artifacts.com/0/home/ubuntu/scikit-learn/doc/_build/html/stable/glossary.html#term-duck-typing